index.js

```diff
@@ -164,6 +210,19 @@ const line = []
 // *** Uncomment the lines below to test
 // console.log("%cScope & Closures - Question 2", "color: red")
 
+
+function ticketNumberGeneratorFunc(){
```

Comment:
Very nice! Great use of closures. You can also actually return the function on line 218 to condense this even more:

```javascript
function ticketNumberGeneratorFunc(){
 let ticketNum = 0
  return function takeATicketNumber(newLine){
    ticketNum += 1
    line.push(ticketNum)
    return `Welcome, You are ticket number ${ticketNum}`
  }
}
```